sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -48,18 +63,22 @@ export class ApplicationOfferingChangeRequestControllerService {
    * @param options method options:
    * - `studentId`: student id for student authorization.
    * - `locationId`: location id for institution authorization.
-   * - `applicationOfferingDetails`: boolean true indicates that only the required application offering details for the student are requested.
+   * - `applicationOfferingDetails`: boolean true indicates that the required application offering details for the institution are requested.
+   * - `applicationOfferingDetailsReview`: boolean true indicates that the required application offering details for the ministry are requested.
    * @returns application offering change request.
    */
   async getApplicationOfferingChangeRequest(
     applicationOfferingChangeRequestId: number,
-    options: {
+    options?: {
       studentId?: number;
       locationId?: number;
       applicationOfferingDetails?: boolean;
```

Comment:
Not part of the PR but option `applicationOfferingDetails` also look very confusing here.